src/pattern-library/components/patterns/navigation/PointerButtonPage.js

```diff
@@ -0,0 +1,141 @@
+import { PointerButton } from '../../../../next';
```

Comment:
Much of the structure of `*Page` components are boilerplate; they should be reviewed for their content, not their code, roughly speaking.